utils/config-utils.js

```diff
@@ -1,7 +1,6 @@
-import devtools from '@vue/devtools';
 import { version } from '_/package.json';
 
-export function initCuttleGlobals(app) {
+export async function initCuttleGlobals(app) {
```

Comment:
I think if we do this we should also look to callers of this function and make them `await` it so that we avoid race condition shenanigans